export/build

```diff
@@ -83,7 +83,7 @@ def highlight_and_texify(text):
     return highlight(text)
 
 
-def make_header(header):
+def make_header(header, processed_fn):
```

Comment:
`processed_fn` is a string, right? The name `*_fn` makes it seem like it's a function

Reply:
I guis this may be confusing for Haskell users & co :slightly_smiling_face:. Changing to `filename`.